package.json

```diff
@@ -22,9 +22,9 @@
     "caseless": "^0.11.0",
     "clone": "^2.1.0",
     "deckardcain": "^0.3.2",
-    "fury": "^2.1.0",
-    "fury-adapter-apib-parser": "^0.3.0",
-    "fury-adapter-swagger": "^0.9.7",
+    "fury": "^3.0.0-beta.1",
```

Comment:
Actually, probably not a good idea to do `^` with a beta. I'd pin to the specific beta that you support to prevent any unwanted breaking changes.